mod/admin.php

```diff
@@ -1492,7 +1492,8 @@ function admin_page_site(App $a)
 		'$performance' => L10n::t('Performance'),
 		'$worker_title' => L10n::t('Worker'),
 		'$relay_title' => L10n::t('Message Relay'),
-		'$relocate' => L10n::t('Relocate - WARNING: advanced function. Could make this server unreachable.'),
+		'$relocate' => L10n::t('Relocate Instance'),
+		'$relocate_warning' => L10n::t('Advanced function. Could make this server unreachable.'),
```

Comment:
Remember you have to reference this variable in all `site.tpl` files, including the base template and all the supported themes that override it.

Reply:
Now I made everything bold. Just for the future: Is it possible to use formatting in `L10n` String ? Or should the template var be something like `'$var => '<b>' . L10n::t('A') . '</b> ' . L10n::t('B');' ?
I wanted to keep the formatting in the template.

Reply:
It's okay, you can put HTML tags in the translation strings.